src/test/scala/com/redislabs/provider/redis/rdd/RedisSqlStandaloneSuite.scala

```diff
@@ -47,10 +51,113 @@ class RedisSqlStandaloneSuite extends FunSuite with ENV with BeforeAndAfterAll w
     loadedArr.sortBy(_.name) should be(data.toArray.sortBy(_.name))
   }
 
+  test("overwrite data when it's empty") {
+    // generate random table, so we can run test multiple times and not append/overwrite data
+    val tableName = "person" + UUID.randomUUID().toString.replace("-", "")
+    val df = spark.createDataFrame(data)
+    df.write.format(RedisFormat)
+      .mode(SaveMode.Overwrite)
+      .save(tableName)
+    val loadedDf = spark.read.format(RedisFormat)
+      .load(tableName).cache()
+    loadedDf.show()
+    loadedDf.count() shouldBe df.count()
+    loadedDf.schema shouldBe df.schema
+    val loadedArr = loadedDf.as[Person]
+      .collect()
+    loadedArr.sortBy(_.name) shouldBe data.toArray.sortBy(_.name)
+  }
+
+  test("overwrite data when it's not empty") {
+    // generate random table, so we can run test multiple times and not append/overwrite data
+    val tableName = "person" + UUID.randomUUID().toString.replace("-", "")
+    val df = spark.createDataFrame(data)
+    df.write.format(RedisFormat)
+      .save(tableName)
+    df.write.format(RedisFormat)
```

Comment:
Can we overwrite with other (modified) payload to make sure it's actually overwritten and not ignored?